torch/_inductor/fx_passes/pre_grad.py

```diff
@@ -89,12 +89,11 @@ def fuse_fx(gm: torch.fx.GraphModule, example_inputs):
         gm = permute_matmul_fusion(gm)
 
     # make sure the autograd is disabled.
-    if torch.is_grad_enabled():
+    if torch.is_grad_enabled() or not is_cpu:
```

Comment:
`remove_identify` and `fuse_conv_bn` are device independent. Why adding a `is_cpu` check here?

Reply:
I think this diff hasn't changed the logic of this part. Before the diff of this PR, these 2 passes also only make effect on CPU. Not sure about the background since these 2 passes looks like device independent.